book/02-using-atom/sections/03-interface.asc

```diff
@@ -24,6 +24,20 @@ Each pane has its own ``items'' or files, which are represented by tabs. You can
 
 To close a pane, close all its editors with `cmd-w`, then press `cmd-w` one more time to close the pane. You can configure panes to auto-close when empty in the Settings view.
 
+[[_pending_files]]
+=== Pending Files
+
+When you open a new file by single-clicking in the tree view, it will open in a new tab with an italic title. This indicates that the file is "pending". When a file is pending, it will be replaced by the next pending file that is opened. This allows you to click around the tree view to find that file you know is there but you don't remember the exact name of without opening a dozen tabs and then having to go back and close them all.
```

Comment:
The last sentence here is quite a sentence :stuck_out_tongue_closed_eyes: – maybe it's worth breaking this out into a more formal example?